src/packages/examples-ext/processors/exampleDependenciesBuilder.js

```diff
@@ -36,15 +39,25 @@ module.exports = function exampleDependenciesBuilder (readFilesProcessor, log, g
 
       // traverse deployments, check project dependencies and add local file
       // dependencies to the list of generated documents
-      generateExamplesProcessor.deployments.forEach(function(deployment) {
```

Comment:
Guess, it worth to report non-array input and provide some configuration tips, rather than support both objects and arrays.
And there's a place to improve the code, isn't it? :)